scripts/infra/local-setup.sh

```diff
@@ -2,6 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 rh__ensure_ibm_system_pkgs() {
+    if command -v ibmcloud &> /dev/null; then
```

Comment:
This makes a lot of sense

one question though: if you call the installation script and `ibmcloud` is already installed, will the script reinstall `ibmcloud` or will it crash?

I guess I’m wondering about the CLI tool and how often it updates. If it has frequent updates, then there may be a situation where you want to reinstall the CLI tool so that you can use the latest features.

Reply:
I have `ibmcloud` installed already, so the script never re-installs it.

`ibmcloud` has its own internal update mechanism, so I occasionally run `ibmcloud update`. I can't remember if it prompts for this occasionally.

Reply:
Oh neat! Good to know!!

Reply:
I think I mis-understood your original question, apologies. I tried https://clis.cloud.ibm.com/install/linux again and it does actually re-install binaries. I'm not sure if it installs the latest?